File: PowerCalculator.java
import java.util.Scanner;

public class PowerCalculator {

  public static void main(String[] args) 
  {
    Scanner scanner = new Scanner(System.in);

    System.out.println("Enter the base number: ");
    double baseNumber = scanner.nextDouble();

    System.out.println("Enter the exponentNumber: ");
    double exponentNumber = scanner.nextDouble();

    double power = calculatePower(baseNumber, exponentNumber);

    System.out.println("The power of " + baseNumber + " raised to" + exponentNumber + " is " + power );
  }


  // Power calculation function
  private static double calculatePower(double baseNumber, double exponentNumber) 
  {
    double calculatedResult = 1;

    for (int i = 0; i < exponentNumber; i++) 
    {
      calculatedResult *= baseNumber;
    }

    return calculatedResult;
  }
}

File: power_calculator.py
def calculate_power(base, exponent):
  """Calculates the power of a number.

  Args:
    base: The base number.
    exponent: The exponent.

  Returns:
    The power of the base number raised to the exponent.
  """

  result = 1
  for i in range(expt):
    result *= base
  return result


def main():
  """Prompts the user for two numbers and prints the power of those numbers."""

  base = int(input("Enter the base No: "))
  exponent = int(input("Enter the ex: "))

  power = calculate_power(base, exponent)
  print(f"The power of {base} raised to {exponent} is {power}")


if __name__ == "__main__":
  main()
